docs/examples/url2file.c

```diff
@@ -57,7 +57,7 @@ int main(int argc, char *argv[])
   /* Switch on full protocol/debug output while testing */
   curl_easy_setopt(curl_handle, CURLOPT_VERBOSE, 1L);
 
-  /* disable progress meter, set to 0L to enable and disable debug output */
+  /* disable progress meter, set to 0L to enable debug output */
```

Comment:
Is it even "debug output" ? Maybe just shorten and simplify it to:
~~~c
/* disable progress meter */

Reply:
I feel the same way regarding the _debug output_ reference.
I think _debug output_ was reffering to the progress meter.

I think the information of what 0L does, reversing the action,
might actually be good to still have. As a user, I want to know
what happens if I do things differently and be informed about
the alternatives that exists.

But I can submit another patch with the shortened:
`/* disable progress meter */`

if you'd prefer.

Reply:
... or 

`/* disable progress meter, set to 0L to enable it */`